extensions/cornerstone/src/panels/PanelMeasurement.tsx

```diff
@@ -48,12 +51,25 @@ export default function PanelMeasurementTable({
 
   const additionalFilter = filterAdditionalFinding(measurementService);
 
-  const { measurementFilter: trackedFilter } = measurementFilters;
-  const measurements = displayMeasurements.filter(
-    item => !additionalFilter(item) && trackedFilter(item)
-  );
+  const { measurementFilter } = measurementFilters;
+  const measurements = displayMeasurements
+    .filter(item => !additionalFilter(item) && measurementFilter(item))
+    .reduce((groupedMeasurements, item) => {
+      const displaySet = displaySetService.getDisplaySetByUID(item.displaySetInstanceUID);
+      const key = displaySet.instances[0].StudyDescription;
+
+      if (!groupedMeasurements.has(key)) {
+        groupedMeasurements.set(key, [item]);
+        return groupedMeasurements;
```

Comment:
Create a grouped measurement object that defines the sub-component to render this on rather than making it a simple list.